web-components/src/components/combobox/ComboBox.ts

```diff
@@ -1386,20 +1383,21 @@ export namespace ComboBox {
 
     getSelectAllOption() {
       return html`
-        <li
+        <div
           id="selectAll"
           part="combobox-option"
-          role="option"
           class="md-combobox-option ${classMap(this.listItemOptionMap)}"
           @click=${this.handleSelectAll}
+          tabindex="-1"
```

Comment:
wy wee need  tabindex="-1" for div element?

Reply:
DOne